packages/cli/src/lib/transform.ts

```
@@ -0,0 +1,110 @@
+  transform: string;
+  filename: string;
+  summary: string;
+}[];
+
+function parseErrors(transform: string, output: string): TransformErrors {
+  const errors: TransformErrors = [];
+  const errorRegex = /ERR (.+) Transformation error/g;
+  const syntaxErrorRegex = /SyntaxError: .+/g;
+
+  let match;
+  while ((match = errorRegex.exec(output)) !== null) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)